samples/bootloader/prj.conf

```diff
@@ -0,0 +1,14 @@
+CONFIG_IS_SECURE_BOOTLOADER=y
+CONFIG_BOOT_BANNER=n # Need this to turn off PRINTK (the bootlader has its own)
+CONFIG_PRINTK=n
```

Comment:
What would be the overhead of using the default printk instead of the bootloader-specific one?

Reply:
Nothing IMO, since it is debug only.

That will be done in the future, it requires shuffling of the upstream build scripts, which is out of scope for this release.

Reply:
Yes, the only overhead is work for making this possible. Also a risk that these changes are not accepted upstream in Zephyr